Explorer/Assets/DCL/Notifications/NotificationTypes/INotification.cs

```diff
@@ -0,0 +1,15 @@
+namespace DCL.Notification
+{
+    public interface INotification
+    {
+        string Id { get; }
+        NotificationType Type { get; }
+        string Address { get; }
+        string Timestamp { get; }
+        bool Read { get; }
+
+        public string GetHeader();
+        public string GetTitle();
+        public string GetThumbnail();
```

Comment:
If there won't be a complex logic, I would convert it to public virtual get-property without Get- prefix in the name: "Title", "Header"...

Reply:
There could be some more complex logic, some notifications might have some combinations of different metadata or even more complex processing of texts